libobs/obs.c

```diff
@@ -2552,6 +2590,12 @@ static inline bool obs_context_data_init_wrap(struct obs_context_data *context,
 	if (!context->procs)
 		return false;
 
+	if (uuid && strlen(uuid) == UUID_STR_LENGTH)
+		context->uuid = bstrdup(uuid);
+	/* Only automatically generate UUIDs for sources */
```

Comment:
I personally dislike placing comments between `if` and `else if` portions, but I understand if placing it at the top would be too disconnected.

Reply:
Yeah there didn't really seem to be any "nice" place to put that comment that didn't look even weirder.